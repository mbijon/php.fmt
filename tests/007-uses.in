<?php
use B;
use C;
use A;
new B;
new C;
?>