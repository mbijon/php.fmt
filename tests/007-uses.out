<?php
use B;
use C;
new B;
new C;
?>